content/en/platform/corda/5.0/developing-applications/upgrading/_index.md

```diff
@@ -0,0 +1,51 @@
+---
+date: '2023-05-18'
+title: "Upgrading CorDapps"
+project: corda
+version: 'Corda 5.0'
+menu:
+  corda5:
+    identifier: corda5-upgrading
+    parent: corda5-develop
+    weight: 9000
+section_menu: corda5
+
+---
+
+# Upgrading CorDapps
+
+This topic describes how to upgrade a CorDapp that has already deployed to Corda to a newer version which Corda will interpret as a new version of the same application. Typically you might want to upgrade your CorDapp to fix bugs, create new functionality or change existing functionality. 
+
+Corda must be able to interpret that two versions of an application are to be considered the same application in order to maintain continuity of any state it has preserved internally relating to the operation of that application.
```

Comment:
Very long sentence to have no punctuation. Consider structure, punctuation, or both.